main.go

```diff
@@ -194,7 +194,7 @@ func trackTimestamp(posFile string) (time.Time, string, error) {
 		return time.Time{}, "", fmt.Errorf("No timestamp found")
 	}
 
-	ts, err := time.Parse(time.RFC3339Nano, "2017-05-21T22:49:23.314299+00:00")
+	ts, err := time.Parse(time.RFC3339Nano, line[:32])
```

Comment:
Is it OK if parsing fails occasionally? I think we may still have 2 timestamp formats in the wild (although ideally everything should be this new format!)

https://github.com/Clever/kinesis-to-firehose/blob/master/decode/decode_test.go#L96-L117